chpl/chpl-service/src/main/java/gov/healthit/chpl/app/presenter/CertifiedProductXmlPresenter.java

```diff
@@ -12,6 +13,7 @@
 import org.springframework.oxm.jaxb.Jaxb2Marshaller;
 
 import gov.healthit.chpl.app.resource.DownloadableResourceCreatorApp;
+import gov.healthit.chpl.app.resource.SEDRow;
 import gov.healthit.chpl.domain.CertifiedProductDownloadResponse;
```

Comment:
Remove unused imports. If you ended up just not changing anything in these files at all you can use Eclipse to replace with the latest from HEAD